File: class_6/class_6/app/src/main/java/com/alejouribesanchez/class_6/Main3Activity.java
package com.alejouribesanchez.class_6;

import android.content.Intent;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;
import android.widget.LinearLayout;

public class Main3Activity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main3);

        if (findViewById(R.id.campo_1).isFocusable()) {
            findViewById(R.id.campo_2).requestFocus();
        }

        LinearLayout linearLayout = (LinearLayout)findViewById(R.id.activity_main3);
        linearLayout.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                startActivity(new Intent(Main3Activity.this, Main4Activity.class));
            }
        });

    }
}

File: class_6/class_6/app/src/main/java/com/alejouribesanchez/class_6/Main3Activity.kt
package com.alejouribesanchez.class_6

import android.content.Intent
import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import android.widget.LinearLayout

class Main3Activity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main3)

        if (findViewById(R.id.campo_1).isFocusable) {
            findViewById(R.id.campo_2).requestFocus()
        }

        val linearLayout = findViewById(R.id.activity_main3) as LinearLayout
        linearLayout.setOnClickListener { startActivity(Intent(this@Main3Activity, Main4Activity::class.java)) }

    }
}
